backend/uclapi/libcal/tests.py

```diff
@@ -268,7 +271,11 @@ def test_bookings_personal_information_stripped(self, m):
             f'/libcal/space/bookings',
             {'ids': 1, 'eid': 123, 'token': self.app.api_token}
         )
-        self.assertEqual(response.status_code, 200)
+
+        # Make sure the number of fields in the returned booking that are not in allowed_fields is 0
+        for booking in response.json()['bookings']:
+            self.assertEqual(len([x for x in booking.keys() if x not in allowed_fields]), 0)
+
         for k, v in response.json()['bookings'][0].items():
             self.assertEqual(k, 'leave_me')
```

Comment:
I don't believe this is tested anymore (as in the results is just empty)? Maybe add a seat_name or similar to the returned json?

Reply:
ah thanks, updated!